helm/values.yaml

```diff
@@ -1,42 +1,56 @@
 ---
-# Voxel51 provided you with a voxel51-docker.json file, you can use the following command to create a Pull Secret:
-#  kubectl --namespace your-namespace-here create secret generic regcred \
-#    --from-file=.dockerconfigjson=./voxel51-docker.json --type kubernetes.io/dockerconfigjson
-# If you use the Voxel51 command above your imagePullSecrets would look like:
+# # Voxel51 provided you with a voxel51-docker.json file, you can use the following command to create a Pull Secret:
+# #
+# # ```shell
+# #  kubectl --namespace your-namespace-here create secret generic regcred \
+# #    --from-file=.dockerconfigjson=./voxel51-docker.json --type kubernetes.io/dockerconfigjson
+# # ```
+# #
+# # If you use the Voxel51 command above your imagePullSecrets would look like:
 # imagePullSecrets:
-#  - name: regcred
+#   - name: regcred
 
 secret:
   fiftyone:
-    # These secrets come from Voxel51
+    # Voxel51 provides these secrets
     apiClientId:
     apiClientSecret:
     auth0Domain:
     clientId:
     clientSecret:
     organizationId:
+
     # These secrets come from your MongoDB implementation
     fiftyoneDatabaseName: fiftyone
     mongodbConnectionString: mongodb://username:password@somehostname/?authSource=admin
-    # This secret is a required random string used to encrypt session cookies
-    # Use something like `openssl rand -hex 32` to generate this string
+
+    # This secret is a required random string used to encrypt session cookies.
+    # To generate this string, run
+    #
+    # ```shell
+    # openssl rand -hex 32
+    # ````
+    #
     cookieSecret:
-    # This key is required and is used to encrypt storage credentials in the MongoDB
-    #   do NOT lose this key!
-    # generate keys by executing the following in python:
+
+    # This required key is used to encrypt storage credentials in the database.
+    #   Do NOT lose this key!
+    # To generate this key, run (in python)
     #
+    # ```python
     # from cryptography.fernet import Fernet
     # print(Fernet.generate_key().decode())
+    # ```
+    #
     encryptionKey:
 
 # apiSettings:
-#   Set `dnsName` to expose the API with host-based routing.
-#   See https://helm.fiftyone.ai/docs/expose-teams-api.html for more information.
+#   # Set `dnsName` to expose the API for host-based routing only.
+#   # See https://helm.fiftyone.ai/docs/expose-teams-api.html for more information.
```

Comment:
commenting our comments?

Reply:
When one goes to uncomment the block, the comments become uncommented.